internal/wasmtools/.gitignore

```diff
@@ -0,0 +1 @@
+target
```

Comment:
Should this be more specific? e.g.`internal/wasmtools/target`?

Reply:
this .gitignore is already in the `internal/wasmtools` folder.